style.css

```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: white;
+  font-family: "poppins", sans-serif;
+}
+
+.flex-container {
+  background-image: url("./images/header-shapes-mobile.png"), url("./images/header-background-mobile.png");
+  background-repeat: no-repeat;
+  padding: 114px 25px 114px 25px;
+}
```

Comment:
- [ ] Even though the css class name is `flex-container`, I don't see a FlexBox container here. Kindly apply FlexBox to this section (`headline` section) as well as you did to the `header` section.
![image](https://user-images.githubusercontent.com/44057863/170784503-f2965b2e-3c53-41d8-9ba8-9f74b68eeb23.png)